sdk/src/main/scala/com/horizen/account/api/http/AccountTransactionApiRoute.scala

```diff
@@ -326,7 +327,7 @@ case class AccountTransactionApiRoute(override val settings: RESTApiSettings,
           secret match {
             case Some(secret) =>
 
-              val to = BytesUtils.toHexString(ForgerStakeMsgProcessor.ForgerStakeSmartContractAddress)
+              val to = BytesUtils.toHexString(FORGER_STAKE_SMART_CONTRACT_ADDRESS_BYTES)
```

Comment:
Should we use Java or Scala naming conventions for constants in Scala?

Reply:
I vote for the uppercase way